flow/activities/flowable.go

```diff
@@ -742,3 +743,24 @@ func (a *FlowableActivity) RenameTables(ctx context.Context, config *protos.Rena
 
 	return sfConn.RenameTables(config)
 }
+
+func (a *FlowableActivity) CreateTablesFromExisting(ctx context.Context, req *protos.CreateTablesFromExistingInput) (
+	*protos.CreateTablesFromExistingOutput, error) {
+	dstConn, err := connectors.GetCDCSyncConnector(ctx, req.Peer)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get connector: %w", err)
+	}
+	defer connectors.CloseConnector(dstConn)
+
+	// check if destination is snowflake, if not error out
+	if req.Peer.Type != protos.DBType_SNOWFLAKE {
+		return nil, fmt.Errorf("rename tables is only supported for snowflake")
```

Comment:
error should be some thing like `create tables from existing is only supported on snowflake`.